src/source/file.js

```diff
@@ -1,9 +1,9 @@
-import { read, open, close } from 'fs';
+import fs from 'fs';
```

Comment:
Using a `default` import makes it easier for bundlers to resolve to an "empty" module.